src/locale.js

```diff
@@ -1,13 +1,19 @@
+import { merge } from 'lodash/fp'
```

Comment:
I went for lodash.merge because it'll merge deeply, so the user can change just one entry and let the rest default for the current ones.